test/parallel/test-child-process-exec-abortcontroller-promisified.js

```diff
@@ -10,12 +10,9 @@ const invalidArgTypeError = {
   name: 'TypeError'
 };
 
-let waitCommand = '';
-if (common.isWindows) {
-  waitCommand = 'TIMEOUT 120';
-} else {
-  waitCommand = 'sleep 2m';
-}
+const waitCommand = common.isLinux ?
+  'sleep 2m' :
+  'node -e "setInterval(()=>{}, 99)"';
```

Comment:
Maybe ${process.execPath}? Also, @Trott would 99 be long enough to be non-flaky on slower machines?

Reply:
> would 99 be long enough to be non-flaky on slower machines?

I don't think this is a concern because I'm using `setInterval`.

Reply:
Yes, you're correct - I noticed that after I wrote my comment (removed it from my commend, but I was too late)...

Reply:
It's possible that `node` isn't in the PATH and so `process.execPath` may still be more robust?